frontend/styles/home.css

```
@@ -0,0 +1,73 @@
+    width: 15px;
+    transition: all .3s ease-out 0s;
+    transform: translateY(-1px);
+}
+
+.arrow-btn:hover svg {
+    color: var(--color-text-dark); 
+    transform: translateX(3); 
+    transform-origin: left center; 
+}
+
+@media (max-width: 1500px) {}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)